components/server/src/server.rs

```diff
@@ -1039,17 +1037,13 @@ where
                 Box::new(BackupStreamConfigManager(backup_stream_worker.scheduler())),
             );
 
-            let etcd_cli = LazyEtcdClient::new(
-                self.config.pd.endpoints.as_slice(),
-                ConnectionConfig {
-                    keep_alive_interval: self.config.server.grpc_keepalive_time.0,
-                    keep_alive_timeout: self.config.server.grpc_keepalive_timeout.0,
-                    tls: Arc::clone(&self.security_mgr),
-                },
-            );
             let backup_stream_endpoint = backup_stream::Endpoint::new(
                 node.id(),
-                etcd_cli,
+                PdStore::new(Checked::new(AutoHeader::new(
+                    Arc::clone(&self.pd_client),
+                    pd_client::Source::LogBackup,
+                    self.pd_client.get_cluster_id().unwrap_or_default(),
```

Comment:
how about remove this parameter of cluster id?

Reply:
Done